src/app/helptext/directoryservice/ldap.ts

```diff
@@ -95,6 +71,10 @@ ldap_certificate_placeholder : T('Certificate'),
 ldap_certificate_tooltip: T('Select the LDAP CA certificate. The certificate for the\
  LDAP server CA must first be imported using the System/Certificates menu.'),
 
+ldap_validate_certificates_placeholder: T('Validate Certificates'),
+ldap_validate_certificates_tooltip: T('Default value is <samp>True</samp>. Checks server certificates in a TLS session. \
+ Turn off in cases where LDAPs is used.'),
```

Comment:
```suggestion
 Unset when LDAPS is used.'),
```

Reply:
We shouldn't advise to unset this. Turning off is not a requirement for LDAPs.

Reply:
Does that apply to AD also?